### DISCUSSION TITLE
Not found issue

### DISCUSSION THREAD

**Discussion by ianrego** [Security-Onion-Solutions/securityonion#7727] [Unsupported Versions]
2.3.110
on-prem
ISO
n/a
1
n/a
yes
no failures, but there is one error wait_for_kratos returning a 404
I can't get to the webui, not sure if something else here
I have installed it but I can't reach the webui
No file in /root/soup.log

![image](https://user-images.githubusercontent.com/83052192/161977829-18f12b8c-1eef-4279-adfb-5573c7dbe384.png)

**Comment by dougburks** [comment]
Please check the logs in `/opt/so/log/kratos/` for additional clues.

**Comment by ianrego** [comment]
Not too many clues there

![image](https://user-images.githubusercontent.com/83052192/162564517-be6d2da9-e456-495b-b0d2-d11d983236ac.png)



Any other ideas?

**Comment by ianrego** [comment]
I've found this with journalctl, will investigate

![image](https://user-images.githubusercontent.com/83052192/162564738-da80888d-2a18-462e-9076-49be69396ba6.png)

**Comment by dougburks** [comment]
Based on the kratos log, it looks like Kratos is running now.

From the journaltcl log, the CROND chdir failure is a [known issue](https://github.com/Security-Onion-Solutions/securityonion/issues/7133) and should be fixed in the next release. It's a minor issue and should not prevent you from logging into the web interface.

Did you run so-allow to allow your IP address to connect?
https://docs.securityonion.net/en/2.3/so-allow.html
